engines/router/.env.development

```diff
@@ -29,6 +29,7 @@ APP_FLUENTD_TAG=response.log
 # For result logging to Kafka
 APP_KAFKA_BROKERS=localhost:9093
 APP_KAFKA_TOPIC=turing-result-log-test
+APP_KAFKA_SERIALIZATION=json
```

Comment:
Does `APP_KAFKA_SERIALIZATION_FORMAT` sound reasonable?
I think it conveys the meaning better

Reply:
Sure, updated it.